packages/x-charts/src/hooks/useTicks.ts

```diff
@@ -2,21 +2,40 @@ import * as React from 'react';
 import { D3Scale } from '../models/axis';
 import { isBandScale } from '../internals/isBandScale';
 
-export type TickParams = {
-  maxTicks?: number;
-  minTicks?: number;
-  tickSpacing?: number;
-};
+export interface TickParams {
+  /**
+   * Maximal step between two ticks. For time axis, value should be provided in ms.
```

Comment:
Are these props only used for time data ?
If you have unit-less data, it does work ?
If it does, we should probably specify "should be provided in ms when using time data"

Reply:
Yes, it should work for all continuous scales.